7_deploy_osh/play.yml

```diff
@@ -29,11 +29,11 @@
         - developer_mode
         - build_osh_images | default('False') | bool
 
-- hosts: osh-deploy-cp
+- hosts: caasp-workers
   gather_facts: yes
   any_errors_fatal: true
   roles:
-    - role: suse-osh-setup
+    - role: setup-caasp-workers
```

Comment:
I renamed the role to explain clearly what it's supposed to do, to which kind of target nodes. We can rename again later if we want to reuse the role based on different groups (we probably shouldn't though).